test-apps/display-test-app/package.json

```diff
@@ -31,17 +31,18 @@
     "test": "",
     "cover": "",
     "copy:config": "internal-tools copy-config",
-    "build:ios-backend": "npm run -s build:backend && webpack --config ./ios/backend.config --env outdir=./lib/ios --env entry=./lib/backend/MobileMain.js --env bundlename=main",
+    "build:ios-backend": "npm run -s build:backend && webpack --config ./mobile_backend.config --env outdir=./lib/ios --env entry=./lib/backend/MobileMain.js --env bundlename=main",
```

Comment:
why not reuse the same mobile backend output for both ios and android instead of both creating the same thing in different dirs?

Reply:
That sounds like a reasonable idea, I'll investigate what other changes that would require.